src/pages/academy/components/utility.ts

```diff
@@ -1,3 +1,42 @@
+import React from 'react'
 import { isBrowser } from 'common/utility'
+import { DerivStore } from 'store'
 
 export const url = isBrowser() ? window.location.href : ''
+
+export const addScriptForCIO = () => {
+    const { is_eu_country } = React.useContext(DerivStore)
+    const site_id = process.env.GATSBY_ENV_CIO_SITE_ID
+
+    let cio_url = 'https://assets.customer.io/assets/track.js'
+    if (is_eu_country) {
+        cio_url = 'https://assets.customer.io/assets/track-eu.js'
+    }
+
+    const addScript = (settings) => {
+        const script = document.createElement('script')
+        const { async, text, src, id } = settings
+
+        if (async) script.async = settings['async']
+        if (text) script.text = settings['text']
+        if (src) script.src = settings['src']
+        if (id) script.id = settings['id']
+        document.body.appendChild(script)
+    }
```

Comment:
This should use the existing `addScript` on `src/common/utility.js`

Reply:
okay @prince-deriv